### ISSUE TITLE
Reproducing audio bug 

### ISSUE THREAD

**Issue by afa7789** [tigase/beagle-im#146]
**Describe the bug**
When I click on the play button of an audio nothing happens.

I can only reproduc audio by clicking on it, and it opens apple music is that right ?
**To Reproduce**
1. Send an audio to someone
2. try to play it

**Expected behavior**

click on play button and listen to it.

**Screenshots**
https://github.com/tigase/beagle-im/assets/26887703/eb26123e-ee28-4662-95ad-38dc6701e29b

**Desktop (please complete the following information):**
 - OS: macOS newer

**Additional context**
Is this the correct behavior ?

**Comment by hantu85** [comment]
What is the macOS version that you are using? On macOS Ventura it correctly plays the file directly in Beagle IM.

**Comment by afa7789** [comment]
@hantu85 I installed it with brew + cask, could it be it ?

Ventura 13.4

**Comment by hantu85** [comment]
No, that would not cause this issue. But now, as I've rewatched your screen capture I understand what is happening.

You did not click on the 'play' button, but on 'open' button, and open action should open received audio file in the installed on the  device audio player - and that is done. However Apple Music fails to play - why? Only Apple knows.

To play audio file in Beagle, click on round button with triangle one the right side. Only clicking in this area will play the file.

**Comment by afa7789** [comment]
Nono, I clicked on the round button (play ) with triangle.

I could reproduce 1 audio, but it was the one sent to me.

I could thin down the problemn. 

IF I send the audio, I can't play it, but If I receive the audio, or sent from other clients it count as downloaded and I can play it.

**Comment by hantu85** [comment]
I think that the file (after being sent) is lost so Beagle is unable to play it  and Apple Music as well (as it cannot find a file). I'll look into that.

**Comment by afa7789** [comment]
okay.

but apple music could find the file.

**Comment by hantu85** [comment]
If Apple Music could find the file then Beagle IM exposing this file should as well. Moreover, the code for playing incoming and outgoing messages is the same, so I do not have any ideas why this would randomly not work.

**Comment by afa7789** [comment]
@hantu85 , is there a way we can go over this together ?
I can share screen, or even try to build it locally and we pair programn, I believe this is worth looking into .
since it's "the best" xmpp  messenger for Mac.
